docs/Configuration/Configuration.md

```diff
@@ -0,0 +1,149 @@
+
+#### SharePoint settings
+![Image of Logic Console Create Document Generation Configuration - SharePoint Settings](./assets/image005.jpg)
+
+`Logic Document Generation` requires a valid SharePoint Client Credential and a Group name to retrieve templates from it.  
+**The Client Credential needs to be granted permissions to read documents and the SharePoint instance Group collection.**
+
+- Client Id:  
+_(Required)_ The Client Id from your SharePoint Client Credentials.
+
+- Tenant Id:  
+_(Required)_ The Tenant Id from your SharePoint Client Credentials.
+
+- Group Name:  
+_(Required)_ The SharePoint group name where the templates are stored.
+
+- Client Secret:  
+_(Required)_ The Client Secret from your SharePoint Client Credentials.
+
+#### Azure Blob Storage settings
+![Image of Logic Console Create Document Generation Configuration - Azure Blob Storage Settings](./assets/image006.jpg)
+
+- Container:  
+_(Required)_ Your Azure Blob Storage container name.
+
+- Connection:
+_(Required)_ Your Azure Blob Storage connection string.
+
+- Blob prefix:  
+_(Optional)_ Your Azure Blob Storage blob prefix.
+
+
+
+### Persisting your changes
+After filling out the details for your configuration and its master storage the `Save` button underneath will get enabled, like so:
+
+![Image of Logic Console Create Document Generation Configuration - Completed](./assets/image007.jpg)
+
+Just click the `Save` button and you'll be all set.
+
+
+![Image of Logic Console Create Document Generation Configuration - Edit mode](./assets/image008.jpg)
+
+
+## Configuration entries
+
+In the screenshot above, we can see a section called `Configuration Entries`. 
+
+This grid list all the entries in our hierarchy, starting from the Configuration Master Entry, like the following grid filled with a couple of entries (most of them misconfigured for demo purposes):
+
+![Image of Logic Console Create Document Generation Configuration - Configuration Entries Grid](./assets/image010.jpg)
+
+
+The grid has the following columns:
+
+- Name:  
+The name of the entry.
+
+- Level:  
+The level of the entry in the hierarchy.
+
+- Storage:  
+The type of storage used by the entry.
+
+- Path:
+The hierarchy path of the entry in the hierarchy.
+
+- Status:  
+Indicates whether the connection to the configured storage can be done successfully or not.  
+Hovering the pointer over the status badge will provide further details about the connection status when possible.  
+
+- Actions:
+
+  - Edit entry (pencil icon):  
+  Allows you to edit the configuration entry.  
+  It will open a pop-up dialog like so:  
+![Image of Logic Console Create Document Generation Configuration - Edit pop up](./assets/image009.jpg)
+
+  - Add child entry (plus icon):  
+  Adds a child entry to the configuration entry.  
+  It will open a pop-up dialog like so:  
+  ![Image of Logic Console Create Document Generation Configuration - Add pop up](./assets/image009.jpg)
```

Comment:
This is the same image as the Edit pop up.

Reply:
Fixed in 2311c5eccc154b7b0a8a079e6daafee382936454